server/README (2).md

```diff
@@ -0,0 +1,15 @@
+# Questioner-API
```

Comment:
This is a duplicated file. Please look into it

Reply:
done